skt/kernelbuilder.py

```diff
@@ -71,7 +71,8 @@ def clean_kernel_source(self):
         logging.info("cleaning up tree: %s", args)
         subprocess.check_call(args)
 
-    def prepare(self):
+    def prepare_kernel_config(self):
```

Comment:
It would be nice give a reason or intent for the rename in the commit message.

Not necessary for an ACK from me, but I would be glad to see that in the future.